packages/lib-extensions/src/extensions/core/actions.ts

```diff
@@ -1,6 +1,6 @@
+import type { CodeRef, Extension } from '@openshift/dynamic-plugin-sdk';
```

Comment:
This looks good - do not change it. It will prevent us from breaking consumers of the SDK today 👍 

 Just a longer term question to pick your brain @vojtechszocs  - do you eventually think we'd switch to pull the extensions directly through the `@openshift/dynamic-plugin-sdk-extensions` package and not `lib-core`?

Reply:
If you mean the `CodeRef` and `Extension` types, these are integral to the core library, which "owns" these types.

I'd keep them there and have `lib-extensions` peer depend on `lib-core` as proposed in this PR.

An alternative would be to bundle (copy) any types from `lib-core` in the extension package's `d.ts` bundle, not sure if that's the right approach as it would effectively duplicate the type definitions.

Reply:
To clarify, by integral I mean core APIs like `useExtensions` and `useResolvedExtensions.ts` depend on these types.